release/scripts/release-pre.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
For systems where `bash` is not located in `/bin`:

```suggestion
#!/usr/bin/env bash
```

Reply:
Done.